build.sbt

```diff
@@ -2058,9 +2058,10 @@ lazy val `std-database` = project
     Compile / packageBin / artifactPath :=
       `database-polyglot-root` / "std-database.jar",
     libraryDependencies ++= Seq(
-      "org.netbeans.api" % "org-openide-util-lookup" % netbeansApiVersion % "provided",
-      "org.xerial"       % "sqlite-jdbc"             % sqliteVersion,
-      "org.postgresql"   % "postgresql"              % "42.4.0"
+      "org.graalvm.truffle" % "truffle-api"             % graalVersion       % "provided",
```

Comment:
We need GraalVM dependency for:
- importing `Value` type - we need to return `Value` not `Object`, as `Object` will cause a polyglot conversion that loses Enso warnings,
- accessing `TruffleLogger` to log that a maintenance operation failed.

Reply:
You cannot use `truffle-api` in standard libraries. Try `sdk`.

Reply:
`truffle-sdk` [does not seem to exist](https://mvnrepository.com/artifact/org.graalvm.truffle)

```
[warn]  Note: Unresolved dependencies path:
[error] stack trace is suppressed; run 'last common-polyglot-core-utils / update' for the full output
[error] (common-polyglot-core-utils / update) sbt.librarymanagement.ResolveException: Error downloading org.graalvm.truffle:truffle-sdk:22.3.1
[error]   Not found
[error]   Not found
[error]   not found: C:\Users\progr\.ivy2\localorg.graalvm.truffle\truffle-sdk\22.3.1\ivys\ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/graalvm/truffle/truffle-sdk/22.3.1/truffle-sdk-22.3.1.pom
```